roles/postgresql/tasks/postgresql_yum6.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: yum |  Setup PostgreSQL repository for {{ ansible_distribution }}
+  become: yes
+  yum:
+    name: https://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-3.noarch.rpm
+    state: installed
+  when: ansible_distribution == "CentOS"
+  tags: [db, postgresql]
+
+- name: yum | Setup PostgreSQL repository for {{ ansible_distribution }}
+  become: yes
+  yum:
+    name: https://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-redhat94-9.4-3.noarch.rpm
+    state: installed
+  when: ansible_distribution == "RedHat"
+  tags: [db, postgresql]
+
+- name: yum | Install postgresql dependency
+  become: yes
+  yum:
+    name: libselinux-python
+    state: installed
+  tags: [db, postgresql]
+
+- name: yum | Install postgresql94
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: installed
+    update_cache: yes
+  with_items:
+    - postgresql94-server
+    - postgresql94-contrib
+    - postgresql94-devel
+  tags: [db, postgresql]
+
+- name: yum | Initialize Postgresql-9.4
+  become: yes
+  command: service postgresql-9.4 initdb
+  tags: [db, postgresql]
```

Comment:
Same, this command needs a "file lock" to ensure that we run `initdb` only once per system and don't repeat it.
_OR_ it's possible to run `init` only when previous block "changed", eg. only after install.

See: https://raymii.org/s/tutorials/Ansible_-_Only-do-something-if-another-action-changed.html